style.css

```diff
@@ -128,3 +131,263 @@ h1 {
   background-position: bottom right;
   background-repeat: no-repeat;
 }
+
+.grid {
+  display: grid;
+  grid-template-rows: auto;
+}
+
+#item1 {
```

Comment:
- [ ] The ```See project``` button should be at the bottom of the project card, please use the propper ``` margin/spacing``` and modify your design so it is as close as much to the Figma design

|Your design|Figma design|
|--|--|
|![Screenshot_20220527_192836](https://user-images.githubusercontent.com/25227022/170770367-bea34071-09f5-41e7-9f4f-2da9436e54c8.png)|![Screenshot_20220527_192940](https://user-images.githubusercontent.com/25227022/170770386-10e01068-fad8-4558-9779-1e10f2c1c252.png)|